lbz/authz/authorizer.py

```diff
@@ -148,7 +147,7 @@ def restrictions(self) -> dict:
         return {"allow": self.allowed_resource, "deny": self.denied_resource}
 
     @staticmethod
-    def sign_authz(authz_data: dict, private_key_jwk: dict) -> str:
+    def sign_authz(authz_data: dict, private_key_jwk: dict) -> Union[str, Any]:
```

Comment:
That is not true, you have here only one return statement in this method and it is declared to always return a string 😉